Add Manage Identity Attributes section (#5017)
wso2/docs-is: en/includes/guides/users/attributes/manage-attributes.md

    Only custom attributes can be deleted.

## Manage Identity Attributes

> By default, identity claim values are stored in the JDBC datasource configured in the `deployment.toml` file. If required, you can configure WSO2 IS to store the claim values in another user store as well.

Reviewer: Is this doc for IS only or for both IS and asgardeo? if this is for both, we may need to use placeholders for the product name

Author: Only applicable for IS. however, display content only for IS condition needed to be added. d3807d00f7a43810a30ecaedba821d4b0d5b763c

    Only custom attributes can be deleted.

{% if product_name == "WSO2 Identity Server" %}
## Manage Identity Attributes

> By default, identity claim values are stored in the JDBC datasource configured in the `deployment.toml` file. If required, you can configure WSO2 IS to store the claim values in another user store as well.

Reviewer: ```suggestion
By default, identity claim values are stored in the JDBC datasource configured in the `deployment.toml` file. If required, you can configure WSO2 Identity Server to store the claim values in another user store as well.
```

> 1.  Open the `<IS_HOME>/repository/conf/deployment.toml` file and add the following configuration to change the `identity_datastore` property value to `IdentityDataStoreService`. This service, functioning as an OSGi service, is dedicated to accessing IdentityDataStores.

Reviewer: ```suggestion
1.  Open the `<IS_HOME>/repository/conf/deployment.toml` file and add the following configuration to change the `identity_datastore` property value to `IdentityDataStoreService`. This service, functioning as an OSGi service, is dedicated to accessing Identity data stores.
```

```
[identity_datastore]
datastore_type = "<Name of the identityDataStore class>"
```

!!! Note

>     The class name configuration for the identity data store has been separated from the listener configurations. If you're using a custom `data store` class, it's crucial to update your `deployment.toml` file to reflect this choice. This update ensures that your custom class will override the default configuration.

Reviewer: ```suggestion
    The class name configuration for the identity data store has been separated from the listener configurations. If you're using a custom data store class, it's crucial to update your `deployment.toml` file to reflect this choice. This update ensures that your custom class will override the default configuration.
```
